CalibTracker/SiPixelLorentzAngle/python/SiPixelLorentzAnglePCLWorker_cfi.py

```diff
@@ -4,13 +4,17 @@
 SiPixelLorentzAnglePCLWorker = DQMEDAnalyzer(
     "SiPixelLorentzAnglePCLWorker",
     folder = cms.string('AlCaReco/SiPixelLorentzAngle'),
+    notInPCL = cms.bool(False),
     fileName = cms.string('testrun.root'),
+    newmodulelist = cms.string('newmodule.txt'),
```

Comment:
can the list of modules be fully specified in the configuration as a vector of strings instead of reading them from another file?